modules/main/pom.xml

```diff
@@ -43,8 +43,8 @@
 
         <application.jersey.provider.paths>com.intuit.wasabi.api;com.codahale.metrics.jersey</application.jersey.provider.paths>
         <application.jersey.pojo.enabled>true</application.jersey.pojo.enabled>
-        <application.jersey.request.filters>com.sun.jersey.api.client.filter.LoggingFilter</application.jersey.request.filters>
```

Comment:
@abhishekjain88 Let's make sure we test that there's no loss of functionality due to removing this filter in both requests & responses: com.sun.jersey.api.client.filter.LoggingFilter